src/main/java/org/broadinstitute/hellbender/tools/spark/sv/BreakpointClusterer.java

```diff
@@ -63,12 +62,35 @@ private void buildTree( final Iterator<BreakpointEvidence> evidenceItr ) {
 
     private boolean hasEnoughOverlappers( final SVInterval interval ) {
         final Iterator<SVIntervalTree.Entry<List<BreakpointEvidence>>> itr = evidenceTree.overlappers(interval);
-        int count = 0;
+        final SVIntervalTree<List<BreakpointEvidence>> targetIntervalTree = new SVIntervalTree<>();
+        int evidenceIntervalOverlapCount = 0;
         while ( itr.hasNext() ) {
-            count += itr.next().getValue().size();
-            if ( count >= minEvidenceCount ) {
+            final List<BreakpointEvidence> evidenceForInterval = itr.next().getValue();
+            evidenceIntervalOverlapCount += evidenceForInterval.size();
+            if ( evidenceIntervalOverlapCount >= minEvidenceCount ) {
                 return true;
             }
+            for (BreakpointEvidence evidence : evidenceForInterval) {
+                if (evidence.hasDistalTargets()) {
```

Comment:
I'm now building up the target tree as I go through the evidence list but then counting them at the end.